README.md

```diff
@@ -102,21 +102,12 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
```

Comment:
- [ ] First of all, great job on creating your `README` for your project! It's an essential part of any project to provide clear documentation. 👍

  - However, I noticed that there are some placeholder text in your `README`. While it's completely understandable that you used these placeholders as reminders during your development process, they may not provide value to the end users of your project. It's essential to ensure that your README is user-friendly and doesn't confuse anyone who comes across it.

   - To improve the clarity of your `README` and make it more user-focused, I'd recommend removing all the placeholder text. This will help users to understand your project better and find the information they need without any confusion. 📝

Happy coding! 😊🚀🍀